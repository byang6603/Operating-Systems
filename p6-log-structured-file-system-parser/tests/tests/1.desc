Does an ls of the simple LFS image.
