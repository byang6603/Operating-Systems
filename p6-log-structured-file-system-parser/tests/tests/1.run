../bin/parse_lfs ls ../sample_files/simple.img
