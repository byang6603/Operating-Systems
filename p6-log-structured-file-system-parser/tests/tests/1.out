/
           0 1988-11-26 03:04:09 UTC -rwxr-xr-x    a.bin   
           0 1973-09-03 10:19:57 UTC -rwxrwxrwx    all.dat 
           0 1988-04-02 21:11:08 UTC -rw-------    bar.txt 
           0 1992-10-24 00:09:46 UTC -rw-------    foo.txt 
